File: Matrix/matrix.py
class Matrix:
	def __init__(self, height, width):
		self.heigth = height
		self.width = width
		self.matrix = self.generate()
		
	def call(self):
		print(self.matrix)

	def generate(self):
		matrix = []
		for i in range(self.heigth):
			temp = []

			for j in range(self.width):
				temp.append(j)

			matrix.append(temp)
		return matrix

if __name__ == '__main__':
	Matrix(4, 5).call()

File: Matrix/matrix.js
class Matrix {
  constructor(heigth, width) {
    this.width = width;
    this.heigth = heigth;
    this.matrix = [];
    this.generate()
  }

  call() {
    console.log(this.matrix)
  }

  generate() {
    for (let i = 0; i < this.heigth; i++) {
      let temp = [];

      for (let j = 0; j < this.width; j++) {
          temp.push(j)
      }

      this.matrix.push(temp)
    }
  }

}

new Matrix(4, 5).call()
